.rubocop.yml

```diff
@@ -4,6 +4,7 @@ AllCops:
 
 Layout/LineLength:
   Max: 100
+  AllowedPatterns: ['^(\s*#)']
```

Comment:
What does this do?

Reply:
>  `AllowedPatterns: ['^(\s*#)']`

Allows long comments over the max length value including if the comment is indented with whitespace